### PR TITLE
Tox improvements (flake8) and cleanup

### PR DESCRIPTION
**Pull request by sbesson**
While reviewing https://github.com/ome/omero-web/pull/38, I was surprised that the missing imports were not picked up via a minimal check. Executing the `flake8` on Python 3 suffices to flag additional `long` and `unicode` imports which are probably not covered by integration tests.

This PR makes the following changes:

- adds `flake8` to the tox environment defined by `tox.ini` for execution under `py27` and `py36`
-  fixes a few Python 3 flake8 errors by importing the relevant `past.builtins` methods
- cleans up dependencies declared in `tox.ini` redundant with `setup.py`
- uncaps Travis CI to run on Python 3 and removes flake8 (now executed via `tox` under multiple Python environment)

### CODE REVIEW

**Comment by sbesson** [comment]
That's related to the fact that `tox.ini` does not configure `flake8` at the moment. I will look into an extra commit  to enable it. That way `flake8` warnings can be flagged during local development rather than on Travis CI only.

**Comment by sbesson** [comment]
Last commits migrate the `flake8` logic out of `.travis.yml` and into `tox.ini` and perform various cleanups of redundant dependencies. Description updated.

File: tox.ini [hunk 1]
```diff
@@ -7,19 +7,16 @@ requires = pip >= 19.0.0
 
 [testenv]
 deps =
+    flake8==2.4.0
```
Comment (joshmoore) [inline tox.ini]: Does a higher version work on py3?

Comment (sbesson) [reply tox.ini]: I have an in-progress branch to uncap `flake8` (primarily revolving about `except: -> except Exception:` and similar cleanups). If we wanted to go there, I would propose to get this in first and open a follow-up PR.

**Comment by joshmoore** [comment]
Merging proactively so we can get the follow-up in.

**Comment by sbesson** [comment]
Opened https://github.com/ome/omero-web/pull/62 for discussion

**Pull request merged**